integration-tests/test/rpc.spec.ts

```diff
@@ -20,6 +20,8 @@ import {
   TransactionRequest,
 } from '@ethersproject/providers'
 import { solidity } from 'ethereum-waffle'
+import simpleStorageJson from '../artifacts/contracts/SimpleStorage.sol/SimpleStorage.json'
```

Comment:
I know this comes from the patterns in the other integration tests but I recommend doing the following instead:

```
import { ethers } from 'hardhat'

...

const storageFactory = await ethers.getContractFactory('SimpleStorage')
const storage = await storageFactory.connect(env.l2Wallet).deploy()
```